webbpsf/webbpsf_core.py

```diff
@@ -752,6 +767,7 @@ def _get_fits_header(self, result, options):
                                            "[arcmin] Det. pos. in telescope V2,V3 coord sys"), after=True)
         result[0].header["APERNAME"] = (self._detectors[self._detector], "SIAF aperture name")
 
+    @SpaceTelescopeInstrument.combine_docstrings()
```

Comment:
I'd like to suggest moving the docstring decorator out of the `SpaceTelescopeInstrument` class. It could be a standalone method in `utils.py` for instance. My motivation for suggesting this is to keep the namespace of class methods a little cleaner - i.e. it's not helpful for tab-completion on every WebbPSF class instance to include `combine_docstrings`, since this is never a function some user would want to call interactively. So we should hide it behind the curtain, so to speak.

Reply:
@shanosborne I wanted to check in with you on this - did you see my above comment on moving this function aside somewhere it won't show up in tab completion? If you agree with that, we could do that then I can merge this PR

Reply:
Yes - I have seen this and am working on moving it to utils.py. However, for some reason now the docstring doesn't show up when I do shift+tab inside the parentheses in an ipython notebook (nor does it show up when I use the question mark in ipython or the notebook). So I'm working on figuring out why that happens.

Reply:
Update: I fixed things by changing combine_docstrings to be a class decorator